.travis.yml

```diff
@@ -14,23 +14,19 @@ install:
   - npm --production=false prune
 jobs:
   include:
-    - stage: test
-      script:
-      - npm run test
-    - stage: deploy
+    - env: NPM_SCRIPT=lint
       node_js: 8
-      script: npm run build
+    - stage: release
+      node_js: 8
+      env: NPM_SCRIPT=build
       before_deploy:
-      - VPKG=$($(npm bin)/json -f package.json version)
-      - export VERSION=${VPKG}-prerelease.$(date +%Y%m%d%H%M%S)
```

Comment:
Also I think this date format is easier to understand than what we're using on VM, so let's keep that.